### ISSUE TITLE
[master] Bugs for upgrade all with admin language preference spanish (es_CO)

### ISSUE THREAD

**Issue by ccarly** [odoo/odoo#7573]
```
Odoo Server Error
Traceback (most recent call last):
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/http.py", line 539, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/http.py", line 576, in dispatch
    result = self._call_function(**self.params)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/http.py", line 312, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/service/model.py", line 113, in wrapper
    return f(dbname, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/http.py", line 309, in checked_call
    return self.endpoint(*a, **kw)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/http.py", line 814, in __call__
    return self.method(*args, **kw)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/http.py", line 405, in response_wrap
    response = f(*args, **kw)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/addons/web/controllers/main.py", line 968, in call_button
    action = self._call_kw(model, method, args, {})
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/addons/web/controllers/main.py", line 956, in _call_kw
    return getattr(request.registry.get(model), method)(request.cr, request.uid, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/api.py", line 223, in wrapper
    return old_api(self, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/addons/base/module/module.py", line 572, in button_immediate_upgrade
    return self._button_immediate_function(cr, uid, ids, self.button_upgrade, context=context)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/api.py", line 223, in wrapper
    return old_api(self, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/addons/base/module/module.py", line 533, in _button_immediate_function
    registry = openerp.modules.registry.RegistryManager.new(cr.dbname, update_module=True)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/modules/registry.py", line 349, in new
    openerp.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/modules/loading.py", line 331, in load_modules
    force, status, report, loaded_modules, update_module)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/modules/loading.py", line 235, in load_marked_modules
    loaded, processed = load_module_graph(cr, graph, progressdict, report=report, skip_modules=loaded_modules, perform_checks=perform_checks)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/modules/loading.py", line 156, in load_module_graph
    _load_data(cr, module_name, idref, mode, kind='data')
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/modules/loading.py", line 98, in _load_data
    tools.convert_file(cr, module_name, filename, idref, mode, noupdate, kind, report)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/tools/convert.py", line 865, in convert_file
    convert_xml_import(cr, module, fp, idref, mode, noupdate, report)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/tools/convert.py", line 952, in convert_xml_import
    obj.parse(doc.getroot(), mode=mode)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/tools/convert.py", line 815, in parse
    self.parse(rec, mode)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/tools/convert.py", line 818, in parse
    self._tags[rec.tag](self.cr, rec, de, mode=mode)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/tools/convert.py", line 722, in _tag_record
    id = self.pool['ir.model.data']._update(cr, self.uid, rec_model, self.module, res, rec_id or False, not self.isnoupdate(data_node), noupdate=self.isnoupdate(data_node), mode=self.mode, context=rec_context )
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/api.py", line 223, in wrapper
    return old_api(self, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/addons/base/ir/ir_model.py", line 1019, in _update
    model_obj.write(cr, uid, [res_id], values, context=context)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/api.py", line 223, in wrapper
    return old_api(self, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/addons/base/res/res_users.py", line 735, in write
    self.update_user_groups_view(cr, uid, context)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/api.py", line 223, in wrapper
    return old_api(self, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/addons/base/res/res_users.py", line 780, in update_user_groups_view
    view.write({'arch': xml_content})
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/api.py", line 221, in wrapper
    return new_api(self, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/api.py", line 528, in new_api
    result = method(self._model, cr, uid, self.ids, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/addons/base/ir/ir_ui_view.py", line 363, in write
    context)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/api.py", line 223, in wrapper
    return old_api(self, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/api.py", line 345, in old_api
    result = method(recs, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/models.py", line 3729, in write
    self._write(old_vals)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/api.py", line 221, in wrapper
    return new_api(self, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/api.py", line 528, in new_api
    result = method(self._model, cr, uid, self.ids, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/models.py", line 3885, in _write
    result += self._columns[field].set(cr, self, id, field, vals[field], user, context=rel_context) or []
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/osv/fields.py", line 1480, in set
    self._fnct_inv(obj, cr, user, id, name, value, self._fnct_inv_arg, context)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/addons/base/ir/ir_ui_view.py", line 179, in _arch_set
    self.write(cr, uid, ids, data, context=context)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/api.py", line 223, in wrapper
    return old_api(self, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/addons/base/ir/ir_ui_view.py", line 363, in write
    context)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/api.py", line 223, in wrapper
    return old_api(self, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/api.py", line 345, in old_api
    result = method(recs, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/models.py", line 3729, in write
    self._write(old_vals)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/api.py", line 221, in wrapper
    return new_api(self, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/api.py", line 528, in new_api
    result = method(self._model, cr, uid, self.ids, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/models.py", line 3823, in _write
    raise UserError(_("You cannot update the translated value of field '%s'") % flabel)
ParseError: "You cannot update the translated value of field 'Arch Blob'
None" while parsing /home/odoo/runbot/extra/runbot/static/build/82560-master-81bf72/openerp/addons/subscription/security/subcription_security.xml:5, near
<record model="res.groups" id="base.group_tool_user">
        <field name="name">User</field>
        <field name="category_id" ref="base.module_category_tools"/>
    </record>
```

**Comment by AidenAndrews** [comment]
Looks like same issue I have just had. Posting link for Reference. https://github.com/odoo/odoo/issues/7587

**Comment by rco-odoo** [comment]
Fixed by revision f84c375c.
